mne_connectivity/var.py

```diff
@@ -0,0 +1,406 @@
+from mne_connectivity.base import Connectivity, EpochConnectivity
+import os
+from pathlib import Path
+import shutil
+import numpy as np
+import scipy
+from scipy.linalg import sqrtm
+from tqdm import tqdm
+from sklearn.linear_model import Ridge
+
+from .utils import fill_doc
+
+
+@fill_doc
+def vector_auto_regression(
+        data, times=None, names=None,
+        model_order=1, delta=0.0,
+        memmap=True, compute_fb_operator=False,
+        n_jobs=1, model='dynamic', verbose=None):
+    """Compute vector auto-regresssive (VAR) model.
+
+    Parameters
+    ----------
+    data : array-like, shape=(n_epochs, n_signals, n_times) | generator
+        The data from which to compute connectivity. The epochs dimension
+        is interpreted differently, depending on ``'output'`` argument.
+    times : array-like
+        The time points used to construct the epoched ``data``.
+    %(names)s
+    model_order : int | str, optional
+        Autoregressive model order, by default 1.
+    delta : float, optional
+        Ridge penalty parameter, by default 0.0
+    memmap : bool
```

Comment:
if you keep it memmap param should be after n_jobs in the list of params

Reply:
Removed.